src/main/java/pages/HomePage.java

```diff
@@ -47,6 +51,16 @@ public void searchText(String textForSearch) {
         searchButton.click();
     }
 
+    public void clickTheAllButton() {
+        allButton.click();
+    }
+
+    public void selectCategory() {
+        smartHomeButton.click();
+        new WebDriverWait(driver, Duration.ofSeconds(10)).until(ExpectedConditions.elementToBeClickable(smartHomeLightingButton));
+        smartHomeLightingButton.click();
```

Comment:
As discussed, fix this method to be more general and give some informative name to method.